asyncscope.md

```diff
@@ -2393,7 +2389,41 @@ creates an asynchronous operation (__[async.ops]__) that, when started:
 
 ## `execution::spawn`
 
-spec here
+Add the following as a new subsection immediately after __[exec.nest]__:
+
+::: add
+__`std::execution::spawn` [exec.scope.spawn]__
+
+[1]{.pnum} `spawn` associates the given input sender with the given token's async scope and, on success, eagerly starts the input sender.
+
+[2]{.pnum} The name `spawn` denotes a customization point object. For subexpressions `sndr`, `token`, and `env`, let `Sndr` be 
+`decltype((sndr))`, let `Token` be `decltype((token)), and let `Env` be `decltype((env))`. If `sender<Sndr>` or `async_scope_token<Snder>` is false, 
+the expression `spawn(sndr, token, env)` is ill-formed. 
+
+// TODO: when/how do I specify that env is optional?  Or do we get the env from sender-in?
+
+`[3]{.pnum} The exposition-only class template `impls-for` ([exec.snd.general])  is specialized for `spawn_t` as follows:
```

Comment:
I think _`impls-for`_ is a helper for defining a sender and its operation state, so I don't think we need it for `spawn`; instead, we need to define the _`spawn-receiver`_ and _`spawn-env`_ expo-only types and explain what `spawn` does with them.